packages/cw-orch-core/src/environment/cosmwasm_environment.rs

```diff
@@ -88,3 +91,11 @@ pub trait TxHandler: ChainState + Clone {
         chain
     }
 }
+
+pub trait WasmCodeQuerier: CwEnv {
+    fn get_contract_hash(&self, code_id: u64) -> Result<String, <Self as TxHandler>::Error>;
```

Comment:
FYI: https://rust-lang.github.io/api-guidelines/naming.html#:~:text=Getter%20names%20follow%20Rust%20convention,for%20getters%20in%20Rust%20code.&text=The%20get%20naming%20is%20used,the%20content%20of%20a%20Cell%20.

Do we want to follow these API guidelines? I'd like us to.

We follow them in the TxHander:

```rust
    /// Gets the address of the current wallet used to sign transactions.
    fn sender(&self) -> Addr;

    /// Sets wallet to sign transactions.
    fn set_sender(&mut self, sender: Self::Sender);
```